lib/awful/hotkeys_popup/keys/tmux.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------
+--- tmux hotkeys for awful.hotkeys_widget
+--
+-- @autor nahsi nashi@airmail.cc
+-- @copyright 2017 nahsi
+-- @module awful.hotkeys_popup.keys.tmux
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local tmux_rule_any = {name={"tmux"}}
+for group_name, group_data in pairs({
+    ["tmux: sessions"] = { rule_any=tmux_rule_any },
+    ["tmux: windows"]  = { rule_any=tmux_rule_any },
+    ["tmux: panes"]    = { rule_any=tmux_rule_any },
+    ["tmux: misc"]     = { rule_any=tmux_rule_any },
```

Comment:
Use a simple "rule" here instead of "rule_any", see https://github.com/awesomeWM/awesome/pull/1829/files#r121262482.